lang/ko.js

```diff
@@ -7,31 +7,33 @@ export default {
 	"components.calendar.selected": "선택됨.",
 	"components.calendar.show": "{month} 표시",
 	"components.dialog.close": "이 대화 상자 닫기",
-	"components.form-element.defaultError": "{label}이(가) 유효하지 않습니다.",
+	"components.form-element.defaultError": "{label}이(가) 잘못되었습니다.",
 	"components.form-element.defaultFieldLabel": "필드",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.url.typeMismatch": "",
+	"components.form-element.input.email.typeMismatch": "{label} is not a valid email",
+	"components.form-element.input.number.rangeOverflow": "Number must be lower than {max}",
+	"components.form-element.input.number.rangeUnderflow": "Number must be higher than {min}",
+	"components.form-element.input.url.typeMismatch": "{label} is not a valid URL",
 	"components.form-element.valueMissing": "{label}이(가) 필요합니다.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-error-summary.errorSummary": "There {count, plural, one { was 1 error } other { were { count } errors }} found in the information you submitted",
 	"components.input-date-range.endDate": "종료일",
-	"components.input-date-range.errorBadInput": "",
+	"components.input-date-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-date-range.startDate": "시작일",
 	"components.input-date-time-range.endDate": "종료일",
 	"components.input-date-time-range.startDate": "시작일",
 	"components.input-date-time.date": "날짜",
 	"components.input-date-time.time": "시간",
 	"components.input-date.clear": "지우기",
-	"components.input-date.errorMaxDateOnly": "",
-	"components.input-date.errorMinDateOnly": "",
-	"components.input-date.errorOutsideRange": "날짜는 {minDate}에서 {maxDate} 사이여야 합니다.",
+	"components.input-date.errorMaxDateOnly": "날짜는 {MaxDate} 이전이어야 합니다.",
+	"components.input-date.errorMinDateOnly": "날짜는 {MinDate} 이후여야 합니다.",
+	"components.input-date.errorOutsideRange": "날짜는 {minDate}와 {maxDate} 사이여야 합니다.",
```

Comment:
Both seem to mean "The date must be between {minDate} and {maxDate}." so seems fine